typed-racket-lib/typed-racket/utils/sealing-contract.rkt

```diff
@@ -7,6 +7,7 @@
 (require racket/class
          racket/contract
          racket/match
+         racket/set
```

Comment:
One question -- where exactly are the sets coming from in this code?

Reply:
Good question. The code is using lists, I think these list represent sets (of method/field/init-field names), so I imported `subset?` and `set=?`

Reply:
This file could just ask for ordered lists and use `equal?`

Reply:
Oh, this uses lists as sets via generics? I would prefer to avoid generic dispatch here.

Reply:
Ok, I'll remove the generic dispatch